SafetyGraphics/safetyGraphics
# Save mapping

# Summary

Adds a new tab showing the source code for the current mapping under the settings/mapping tab. This code can be pasted in the R Console to initialize the app with the current mapping. 

This closes #435 #448.

# Test notes

1. Start the app, and make some changes on the data/mapping tab
2. Confirm that the changes are shown on the settings/mapping/summary tab
3. Copy the code from the settings/mapping/code tab
4. Restart the app using the copied code and confirm that the changes the the mapping are still present.
